src/components/landingPageHero.module.scss

```diff
@@ -39,18 +39,17 @@
 @media screen and (max-width: 760px) {
   .hero {
     padding: 4rem 0;
-  }
-
-  .text {
-    padding-left: 2.5rem;
+    .text {
+      padding-left: 2.5rem;
 
-    h1 {
-      font-size: 2.8rem;
-    }
+      h1 {
```

Comment:
I assume these changes are to avoid applying these styles to the rest of the page?

Reply:
I can't remember; I'll include the fix in my next PR